plugins/keycloak-backend/package.json

```
@@ -14,7 +14,9 @@
     "role": "backend-plugin"
   },
   "scripts": {
-    "start": "backstage-cli package start",
+    "start": "concurrently -n 'keycloak,backstage' 'yarn start:keycloak' 'yarn start:dev' 'open http://localhost:8080/admin/master/console/#/janus-realm' 'open http://localhost:7007/catalog/entities'",
+    "start:dev": "backstage-cli package start",
+    "start:keycloak": "podman run -p 8080:8080 -e 'KEYCLOAK_ADMIN=admin' -e 'KEYCLOAK_ADMIN_PASSWORD=admin' -v ./__fixtures__/keycloak-realm.json:/opt/keycloak/data/import/keycloak-realm.json quay.io/keycloak/keycloak:22.0.1 start-dev --import-realm",
```

```suggestion
    "start": "turbo run start:backstage start:fixtures",
    "start:backstage": "backstage-cli package start",
    "start:fixtures": "podman run -p 8080:8080 -e 'KEYCLOAK_ADMIN=admin' -e 'KEYCLOAK_ADMIN_PASSWORD=admin' -v ./__fixtures__/keycloak-realm.json:/opt/keycloak/data/import/keycloak-realm.json quay.io/keycloak/keycloak:22.0.1 start-dev --import-realm",
```

You'll also need to add the following to `turbo.json`.

```json
{
  ...
  "pipeline": {
    "start": {
      "cache": false,
      "persistent": true,
      "dependsOn": ["^build"]
    },
    "start:backstage": {
      "cache": false,
      "persistent": true
    },
    "start:fixtures": {
      "cache": false,
      "persistent": true
    },
    ...
  }
}
```

I changed the scripts to `start:backstage` and `start:fixtures` to be more generic so other plugins can use this template. The main benefits to this would be an easier template to copy and less dependencies.

---

Thank you! 👍 Nice job. I'll apply this change.

---

Nice... I've managed to find an even less invasive solution. 🙂 PTAL

---

I'll do it in a separate PR 🙂